src/Network/NWProtocolOptions.cs

```diff
@@ -38,25 +31,28 @@ public class NWProtocolOptions : NativeObject {
 		[DllImport (Constants.NetworkLibrary)]
 		static extern IntPtr nw_tls_create_options ();
 
+		[Obsolete ("Use the 'NWProptocolTlsOptions' class methods and constructors instead.")]
```

Comment:
typo -> `NWProptocolTlsOptions` should be `NWProtocolTlsOptions`
need to fix all `[Obsolete] messages